tests/mqtt-connect-rules/test.yaml

```diff
@@ -0,0 +1,60 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  files:
+    - rust/src/mqtt/parser.rs
```

Comment:
We can just do with the first one

Reply:
Hmm, the first one? I'd say that the presence of `src/detect-mqtt-connect-protocol-string.c` should be enough to indicate that the feature can be tested. I'll leave that one in.

Reply:
Indeed. I meant just keep `src/detect-mqtt-connect-protocol-string.c`